buildSrc/src/main/java/BaseProjectConfig.kt

```diff
@@ -83,6 +83,7 @@ internal fun BaseAppModuleExtension.configureAndroidApplicationOptions(project:
   buildFeatures {
     viewBinding = true
     buildConfig = true
+    compose = true
```

Comment:
If we're setting `compose` properties here then the repository configuration should be moved here too.

Reply:
The problem with moving `compose` properties here is that we won't be able to access the compose snapshot version present in the `libs.versions.toml`. I can move this to `app/build.gradle.kts` to keep all the stuff at one place. What are your thoughts on that?

Reply:
That works too